### PR TITLE
add hammerdb automation

### PR DESCRIPTION
**Pull request by SaitTalhaNisanci**

### CODE REVIEW

File: hammerdb/build-and-run.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# fail if trying to reference a variable that is not set.
+set -u
+# exit immediately if a command fails
+set -e
+# echo commands
+set -x
```
Comment (JelteF) [inline hammerdb/build-and-run.sh]: Should add `set -o pipefail` too, also to other shell scripts.

File: hammerdb/build-and-run.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```
Comment (JelteF) [inline hammerdb/build-and-run.sh]: Please install `shellcheck` and fix/explicitely ignore the warnings it gives. Bash is nice, but very brittle if you don't follow shellcheck.

**Comment by onderkalaci** [comment]
I'm in favor of merging this and adding some documentation on how to use it. This is a great test to have during the release tests.

@SaitTalhaNisanci what do you think?

**Comment by SaitTalhaNisanci** [comment]
Yeah it sounds good, this is not the PR we want to merge though. I will create a PR with separate commits based on the latest branch ( talha_test)

**Pull request closed without merging**